cnxeasybake/scripts/main.py

```diff
@@ -45,6 +52,9 @@ def main(argv=None):
                         help='Stop baking just before given pass name')
     parser.add_argument('-d', '--debug', action='store_true',
                         help='Send debugging info to stderr')
+    parser.add_argument('-l', '--lcov',
+                        type=argparse.FileType('w'),
+                        help="lcov output coverage file")
```

Comment:
lcov seems a term-of-art, is it obvious what it is? Should this be `-c --coverage` instead?